packages/engine/src/avatar/components/AvatarIKComponents.ts

```diff
@@ -68,13 +67,14 @@ export const AvatarIKTargetComponent = defineComponent({
     const debugEnabled = useHookstate(getMutableState(RendererState).avatarDebug)
 
     useEffect(() => {
-      if (!debugEnabled.value) return
-      const helper = new AxesHelper(0.5)
-      addObjectToGroup(entity, helper)
-      setObjectLayers(helper, ObjectLayers.AvatarHelper)
-      setComponent(entity, VisibleComponent)
-      return () => {
-        removeObjectFromGroup(entity, helper)
+      if (!debugEnabled.value) {
```

Comment:
prefer a cleanup function to remove the component, such that when the AvatarIKTargetComponent is removed, this component is also removed